.github/workflows/IJ.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IC-2022.2, IC-2022.3, IC-2023.1, IC-2023.2, IC-2023.2.1]
```

Comment:
you should replace it with IC-2023.2.3 (released today) regardless of the 2023.3 EAP situation

Reply:
thanks, fixed